packages/@internationalized/message/src/MessageFormatter.ts

```diff
@@ -39,7 +39,14 @@ export class MessageFormatter {
       message = new IntlMessageFormat(msg, this.locale);
       this.cache[key] = message;
     }
+    let varCopy: Record<string, PrimitiveType | T | FormatXMLElementFn<T, string | T | (string | T)[]>> | undefined;
+    if (variables) {
+      varCopy = Object.keys(variables).reduce((acc, key) => ({
+        ...acc,
+        [key]: variables[key] == null ? false : variables[key]
+      }), {});
```

Comment:
instead of copying the object a bunch (potentially), you could just assign. you already created a new object on line 47 to add to
```suggestion
      varCopy = Object.keys(variables).reduce((acc, key) => {
        acc[key] = variables[key] == null ? false : variables[key]
        return acc;
      }, {});
```

Reply:
good point 👍